### DISCUSSION TITLE
Tabs and tabs?

### DISCUSSION THREAD

**Discussion by danielrode** [LazyVim/LazyVim#677] [Q&A] [answered]
What plugin is causing these numbered buttons in the top-right of the screen? How can I disable them?

![image](https://user-images.githubusercontent.com/2867646/234359801-bee453db-ef10-427d-b6cc-0c203d36a60d.png)

The numbered "tabs," if you will, seem to arbitrarily map to the actual tabs that show up on the left; sometimes several of the number buttons map to the same tab/buffer. I do not understand how/why the numbered buttons work, but they seem redundant, so I'd like to get rid of them.

**Comment by dpetka2001** [comment, marked as answer]
The numbered "tabs" that you see on the right are the actual tabs. The ones on the left are "buffers". Maybe you should read more info about Vim/Neovim to understand its logic. You can have many buffers on one tab or you can even spread out your buffers on multiple tabs. Depends on your workflow.

**Reply by danielrode** [reply]
Oh! I understand what is happening now. One of my plugins is using `--remote-tab` to open new files rather than `--remote`, which is creating the mess I am seeing with everything doubling. I know how it fix it now. Thank you!
